test/behavior/floatop.zig

```diff
@@ -697,3 +697,38 @@ test "f128 at compile time is lossy" {
 
     try expect(@as(f128, 10384593717069655257060992658440192.0) + 1 == 10384593717069655257060992658440192.0);
 }
+
+test "f16 literals" {
+    if (builtin.zig_backend != .stage2_c) return error.SkipZigTest; // TODO
```

Comment:
Please at least run the behavior tests with the LLVM backend (which is now the default backend) and don't disable it for the LLVM backend. Look at the `@sqrt` test in this file for a preamble that looks appropriate here.

Reply:
For some reason it was failing on the LLVM backend in the past, it is not the case anymore so I removed the conditional.